seed/views/v3/organizations.py

```diff
@@ -1392,3 +1403,353 @@ def reset_all_passwords(self, request, pk=None):
+        }
+
+        taxlot_state = TaxLotState(**taxlot_details)
+        taxlot_state.save()
+        taxlot_1 = TaxLot.objects.create(organization=org)
+        taxview = TaxLotView.objects.create(taxlot=taxlot_1, cycle=cycle, state=taxlot_state)
+
+        state_ids_to_merge = []
+        property_views = []
+        properties = []
+        ids = []
+        for dic in property_details:
+            state = PropertyState(**dic)
+            state.save()
+            ids.append(state.id)
+
+            # merge
+            for count in range(len(property_details)):
+                if count > 9:
+                    state_ids_to_merge = [state.id]
+                    state_ids_to_merge.append(state.id)
+
+            property_1 = Property.objects.create(organization=org)
+            properties.append(property_1)
+            propertyview = PropertyView.objects.create(property=property_1, cycle=cycle, state=state)
+            property_views.append(propertyview)
+
+            # create labels and add to records
+            new_label, created = Label.objects.get_or_create(color="red", name="Housing", super_organization=org)
+            if state.extra_data.get('Note') == 'Residential':
+                propertyview.labels.add(new_label)
+
+            PropertyAuditLog.objects.create(
+                organization=org,
+                state=state,
+                record_type=AUDIT_IMPORT,
+                name='Import Creation'
+            )
+
+            # Geocoding - need mapquest API (should add comment for new users)
+            geocode = PropertyState.objects.filter(id__in=ids)
+            geocode_buildings(geocode)
+
+        # Create a merge of 2 properties
+        merged_state = merge_properties(state_ids_to_merge, pk, 'Manual Match')
+        match_merge_link(merged_state.propertyview_set.first().id, 'PropertyState')
+
+        # pair a property to tax lot
+        property_id = property_views[0].id
+        taxlot_id = taxview.id
+        pair_unpair_property_taxlot(property_id, taxlot_id, org, True)
+
+        # create column for Note
+        Column.objects.get_or_create(
+            organization=org,
+            table_name='PropertyState',
+            column_name='Note',
+            is_extra_data=True  # Column objects representing raw/header rows are NEVER extra data
+        )
+
+        import_record = ImportRecord.objects.create(name='Auto-Populate', super_organization=org)
+
+        # Interval Data
+        filename = "example-pm-monthly-meter-usage.xlsx"
+        filepath = os.path.dirname(os.path.abspath(__file__)) + "/data/" + filename
+
+        import_meterdata = ImportFile.objects.create(
+            import_record=import_record,
+            source_type="PM Meter Usage",
+            uploaded_filename=filename,
+            file=SimpleUploadedFile(name=filename, content=open(filepath, 'rb').read()),
+            cycle=cycle
+        )
+
+        save_raw_data(import_meterdata.id)
```

Comment:
This starts a background task which might not finish before we return 'success' to the client. Not sure how much of a concern that is for now. (same goes for later call to save_raw_data)